mlflow/runs.py

```diff
@@ -0,0 +1,74 @@
+"""
+CLI for runs
+"""
+import click
+import datetime
+import json
+from mlflow.entities import ViewType
+from mlflow.tracking import _get_store
+from tabulate import tabulate
+
+RUN_ID = click.argument("run_id", type=click.STRING)
+
+
+@click.group("runs")
+def commands():
+    """
+    Manage runs. To manage runs of experiments associated with a tracking server, set the
+    MLFLOW_TRACKING_URI environment variable to the URL of the desired server.
+    """
+    pass
+
+
+@commands.command("list")
+@click.option("--experiment_id", help="Specify the experiment for list of runs.", type=click.INT)
```

Comment:
This should be required (or else default to 0 -- right now it defaults to None). Also, we typically use dashes, so like `--experiment-id`. Probably we can just copy this one:

https://github.com/mlflow/mlflow/blob/d90e017e8cb3fee25168a5f7c3668bcf568bc7c0/mlflow/cli.py#L49-L51